### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by paavanb**
Added files necessary to make the project a python package.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,5 @@
+include LICENSE
+include README.md
+recursive-include django_jsconstants/templates *.py *.html
+recursive-include django_jsconstants/templatetags *.py
```
Comment (robmadole) [inline MANIFEST.in:4]: Is this needed? I would think that we'd get these for free.

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+recursive-include django_jsconstants/tests *.py
```
Comment (robmadole) [inline MANIFEST.in:5]: Ditto.

File: MANIFEST.in (hunk 1 above)
Comment (paavanb) [inline MANIFEST.in:4]: I followed the django tutorial, which had those lines. Also, I found that if I didn't explicitly specify the type, .pyc files would get included as well.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='django-jsconstants',
+    version='1.0',
+    packages=['django_jsconstants',],
+    license='Apache 2.0 license',
+    description='An interface to communicate constants between Python and Javascript.',
```
Comment (robmadole) [inline setup.py]: Let's bat this around a little bit.

"A Django application to allow your JavaScript running in the browser to use constants written in Python."

Does that read?

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    author='Kyruus',
+    classifiers=[
+        'Environment :: Web Environment',
+        'Framework :: Django',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License', 
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
```
Comment (robmadole) [inline setup.py:25]: If we are going to claim this (which we totally should) we need to back it up with tests.

Time for tox? https://pypi.python.org/pypi/tox

**Pull request closed without merging**